src/internals/esp32_string.h

```diff
@@ -1,38 +1,85 @@
 #ifndef TINY_CLIENT_ESP32STRING_H
 #define TINY_CLIENT_ESP32STRING_H
 
-#endif //TINY_CLIENT_ESP32STRING_H
+
```

Comment:
Den her skal flyttest til bunden af det her kode. Grunden til de her infdef bliver skrevet, er så man ikke includer det samme kode to gange. Fx hvis vi kalder "#include <NativeString.h>" i to forskellige filer, så loader vi faktisk ikke det her kode to gange

Reply:
Okay glem det, jeg kan se at det er der på linje https://github.com/Forsteholdet/TinyClient/pull/4/files#diff-360c4915720470665b56a568927f464278a8a2d462264468990684f65115629aR85